### DISCUSSION TITLE
azuracast & ffmpeg server on same vps

### DISCUSSION THREAD

**Discussion by Heimatradio** [AzuraCast/AzuraCast#7002] [Q&A]
i'm running Azuracast on a vps with 6 cpus and 16gb ram.
installed with the docker-install
now my question:
is it possible to run an ffmpeg server for video-streamning on the same vps?
Is Azuracast using nginx or apache?
is there anywhere an step-by-step instruction to setup an video-server on a Azuracast running vps?
greetings and thanks in advance

**Comment by gAlleb** [comment]
There are a few options, depending on what you want. But short answer is `yes`.
Azuracast uses `nginx` in docker.

1. You can run your videoserver on your own `nginx` or `apache` on different ports parallel to Azuracast on default ports (80, 443).
2. You can run Azuracast behind a proxy (e.g. `nginx proxy`) on subdomain and setup ffmpeg server on default ports or vice versa.
3. Finally you can setup video-streaming server inside Azuracast itself with the help of Liquidsoap.
